ui/src/modules/Circles/Comparation/Item/CreateSegments/Percentage.tsx

```diff
@@ -0,0 +1,153 @@
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useEffect, useState } from 'react';
+import { FormProvider, useForm } from 'react-hook-form';
+import Text from 'core/components/Text';
+import get from 'lodash/get';
+import {
+  CIRCLE_STATUS,
+  useCirclePercentage,
+  useSaveCirclePercentage
+} from 'modules/Circles/hooks';
+import Loader from './Loader';
+import Styled from './styled';
+import SliderPercentage from './Slider';
+import { getProfileByKey } from 'core/utils/profile';
+import { Circle } from 'modules/Circles/interfaces/Circle';
+import CirclePercentageList from '../Percentage/CirclePercentageList';
+import AvailablePercentage from '../Percentage/AvailablePercentage';
+import { editingPercentageLimit } from './helpers';
+
+interface Props {
+  id: string;
+  circle: Circle;
+  onSaveCircle: Function;
+  isEditing: boolean;
+}
+
+interface SliderForm {
+  slider: string;
+}
+
+const Percentage = ({ id, circle, onSaveCircle, isEditing }: Props) => {
+  const { name, percentage, deployment } = circle;
+  const methods = useForm<SliderForm>({ mode: 'onChange' });
+  const {
+    handleSubmit,
+    getValues,
+    formState: { isValid }
+  } = methods;
+  const [responseSaveCircle, saveCircle, isSaving] = useSaveCirclePercentage(
+    id
+  );
+  const [responseGetCircles, getFilteredCircles] = useCirclePercentage();
+  const [limitPercentage, setLimitPercentage] = useState<number>(null);
+
+  const onSubmitValue = () => {
+    const authorId = getProfileByKey('id');
+    const rangeValue = getValues('slider');
+    saveCircle({ authorId, percentage: Number(rangeValue), name });
+  };
+
+  useEffect(() => {
+    const sumPercentage = get(
+      responseGetCircles,
+      'content[0].sumPercentage',
+      0
+    );
+    if (sumPercentage > 0) {
+      // Calculate the available percentage based in active circles (all active percentage added is sumPercentage)
+      return setLimitPercentage(100 - sumPercentage);
+    }
+    // if sumPercentage is zero, the total available will be 100%.
+    return setLimitPercentage(100);
+  }, [responseGetCircles]);
+
+  useEffect(() => {
+    getFilteredCircles('', CIRCLE_STATUS.active);
+  }, [getFilteredCircles]);
+
+  useEffect(() => {
+    if (responseSaveCircle) {
+      onSaveCircle(responseSaveCircle);
+    }
+  }, [responseSaveCircle, onSaveCircle]);
+
+  const renderWarningNoPercentageAvailable = () =>
+    !isEditing &&
+    limitPercentage === 0 && (
```

Comment:
This logic is reused, so maybe creating a `const` with this and call in all places will be a good option.

```suggestion
    const isUnable = !isEditing && limitPercentage === 0
```

Reply:
Done